### PR TITLE
implement basic wrapper to fetch the authentication_token for a user

### PR DESCRIPTION
**Pull request by sgershey**
Add new wrapper method to library per:

https://portal.hyperwallet.com/content/api/v3/resources/authentication-token/create

### CODE REVIEW

File: hyperwallet/utils/apiclient.py [hunk 1]
```diff
@@ -107,7 +107,7 @@ def _makeRequest(
                 headers=headers,
                 params=params
             )
-        except:
+        except Exception:
```
Comment (melayev) [inline hyperwallet/utils/apiclient.py]: They might have left this part "catch-all" intentionally as it involves network and OS https://docs.python.org/3.6/library/exceptions.html#base-classes

**Review by melayev — APPROVED** [review]
Looks good to me 👍

File: hyperwallet/utils/apiclient.py (hunk 1 above)
Comment (Lewiscowles1986) [reply hyperwallet/utils/apiclient.py]: BaseException if not Simon, it's the python core one equivalent to this

Comment (Lewiscowles1986) [reply hyperwallet/utils/apiclient.py]:
```suggestion
        except BaseException:
```

**Review by Lewiscowles1986 — APPROVED** [review]
👍

**Pull request merged**